data/kahet/kahet missions.txt

```diff
@@ -2020,3 +2020,352 @@ conversation "end of Patir 2 - on complete"
+			`	You try to come closer, but the compass on your ship suddenly starts spinning randomly! Alarmed, you swiftly reverse course to move away from the facility, and the instrumentation soon returns nominal again after a reboot. The automated outpost is clearly emitting a strong magnetic field, but the connection among all this isn't immediately obvious.`
+			choice
+				`	(Do a close flyby, even if the compass goes haywire.)`
+				`	(Try to search around the facility, keeping a safe distance.)`
+					goto coward
+				`	(Give up and move to the original target.)`
+					goto quarantine
+			`	A brief exploration doesn't turn up anything you hadn't already seen earlier, and all you can do is watch as the mesmerizing flow of Faes'mar in and out of the facility continues undisturbed below you.`
+			label coward
+			`	Just as you're starting to feel this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of precious metal particles.`
+			`	The overall picture has become clearer: the frequent asteroid impacts on a moon like this raise enough meteorite dust to effectively shroud the planet in an extremely sparse atmosphere, which the Builder machines attract and gather so that they can be transported off-planet. It's a slow and cumbersome process, but it's also one that was able to work for tens of thousands of years without maintenance.`
+			label quarantine
+			`	You record a brief log of your thoughts, and move on to your actual destination. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, the map eventually signals you have reached your destination: a plain, unremarkable plateau extending for kilometers in every direction.`
+			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's absolutely nothing here, as if it was never even inhabited in the first place.`
+			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain going just at the right speed, careful not to skew the data with the wrong movement, to allow the instruments to function correctly.`
```

Comment:
```suggestion
			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain going at just the right speed, careful not to skew the data with any abrupt movements, so that the instruments can function correctly.`
```